templates/test/running_table.html.ep

```diff
@@ -48,12 +48,12 @@
                     of <%= "$distri-$version-$flavor.$arch" %>
                 </td>
                 <td class="test">
-		    % if (is_operator) {
-			%= link_to url_for('apiv1_cancel', jobid => $job->id) => (class => 'cancel') => begin
+            % if (is_operator) {
```

Comment:
uhm…

Reply:
so why these indendation change?

Reply:
that's tabs I assume?

Reply:
I see.

@foursixnine Please update the git commit message subject line of "Changes in formatting of the template" to something more sexy. And I guess you need to ensure indendation is correct after the tab replacement

Reply:
So I'll fix that commit message, with the commit description then `Replace tabs with spaces and fix indentation` _fixes the operator indentation too_